scan-command/src/test/resources/test-resources/core-rules/rule_checkpanic.bal

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+function getResult() returns json|error {
+    return {"result": "message"};
+}
```

Comment:
```suggestion
function getResult() returns int|error => 1;
```

Unless specific things are required by the test, let's keep code as simple as possible.